app/controllers/api/v1/packages_controller.rb

```diff
@@ -251,6 +260,10 @@ def stock_serializer
         Api::V1::StockitItemSerializer
       end
 
+      def not_splittable?(qty_to_split)
+        @package.quantity.zero? || @package.singleton_package? || qty_to_split.to_i.zero? || qty_to_split >= @package.quantity || qty_to_split < 1
```

Comment:
@Darshan-Chauhan , could you please make sure code doesnt exceeds 80 chars per line ? Doing that way, its readable and much more understandable.
Also to many conditions, can we split these conditions to seperate methods ?
Thoughts ? 

cc - @namrataukirde

Reply:
@abhinav098 , conditions are already separated out in different method. :)

Reply:
I can split it if it was written in if condition but cant split it here as it's the single line which is returning boolean